src/up.ts

```diff
@@ -39,25 +36,22 @@ const go = async function () {
   // TODO: try `parseSync`
   // @ts-ignore
   if (argv.init) {
-    // If these aren't specified then we want to open a terminal prompt that
-    // will help the user setup their project directory then exit when finished
+    // If init arg is given we want to open a terminal prompt that will
+    // help the user setup their project directory then exit when finished
     await projectBuilder();
     return;
   }
 
+  const opts: Config = {};
+
   // @ts-ignore
```

Comment:
why the ignores here?

Reply:
it is safer to use `as any` in these cases, because you might be ignoring something else entirely

Reply:
Nice! 
I just pushed a change.